dbt/adapters/athena/session.py

```diff
@@ -208,13 +167,20 @@ def start_session(self) -> UUID:
             dict: The session information dictionary.
 
         """
+        description = self.session_description
```

Comment:
concern: I've questioned if the session description should be a property of the session manager. If its not supposed to be then this will have to be modified